client.lua

```diff
@@ -256,20 +274,36 @@ RegisterNetEvent('qb-jewelery:client:alarm', function()
     StopAlarm('JEWEL_STORE_HEIST_ALARMS', true)
 end)
 
-CreateThread(function()
-    while true do
-        if #(GetEntityCoords(cache.ped) - Config.Cabinets[1].coords) < 50 then
-            for i = 1, #Config.Cabinets do
-                if Config.Cabinets[i].isOpened then
-                    local RayFire = GetRayfireMapObject(Config.Cabinets[i].coords.x, Config.Cabinets[i].coords.y, Config.Cabinets[i].coords.z, 1.4, Config.Cabinets[i].rayFire)
-                    SetStateOfRayfireMapObject(RayFire, 9)
+lib.zones.sphere({
+    coords = vec3(Config.Cabinets[1].coords.x, Config.Cabinets[1].coords.y, Config.Cabinets[1].coords.z),
+    radius = 80,
+    --debug = true,
+    onEnter = function()
+        createElectricalBox()
+        CreateThread(function()
+            
```

Comment:
```suggestion
            insideJewerly = true
```
Seems to me like you forgot this

Reply:
Good catch. Corrected.